templates/plug_html_template.tmpl

```diff
@@ -0,0 +1,23 @@
+<html>
+	<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">
+                <center>
+		<h1 style="color: white;">Title Goes Here!</h1>
+		<h2 style="color: white;">Subtitle goes here!</h2>
+		<p style="color: white";>
+                        Informative text goes here
+		</p>
+		<div>
+		<p style="color: white";>
+                        Whatever form/image/other thing you want can go here
+		</p>
+		</div>
+		<div>
+        		<br />
+			<a href="https://github.com/liam-middlebrook/csh-plug">Click here for CSH Plug on GitHub</a>
+		</div>
+		<script>
+ 		<!�� Put whatever script you want here ��>

```

Comment:
Nitpick: If you're in JS you would use `// comment` or `/* comment */` for a comment.

Also please remove this script section.